runtime.js

```diff
@@ -150,66 +150,75 @@ function integers_uint32_to_string(i) {
    return caml_new_string(i.value.toString());
```

Comment:
I think you'll also need a js reviewer for this.

Reply:
As said in [this commit](66b7bea9a2a63d67dc3bfd97026db1608b20a81f), it is imported from Mina. The changes have been reviewed before.

Reply:
Why?